docs/reference/commandline/service_create.md

```diff
@@ -667,59 +667,71 @@ $ docker service create \
 ### Specify service constraints (--constraint)
 
 You can limit the set of nodes where a task can be scheduled by defining
-constraint expressions. Multiple constraints find nodes that satisfy every
+constraint expressions. Constraint expressions can either use a _match_ (`==`)
+or _exclude_ (`!=`) rule. Multiple constraints find nodes that satisfy every
 expression (AND match). Constraints can match node or Docker Engine labels as
 follows:
 
+node attribute       | matches                  | example
+---------------------|--------------------------|-----------------------------------------------
+`node.id`            | Node ID                  | `node.id==2ivku8v2gvtg4`
+`node.hostname`      | Node hostname            | `node.hostname!=node-2`
+`node.role`          | Node role                | `node.role==manager`
```

Comment:
As this is an enum, we should add the values here in the `matches` column: `Node role (manager|worker)`

Reply:
updated; used a `/` instead of `|` because otherwise the markdown renderer might get confused 😓